workshops/fetch_a_hack_clubber/es-xl.md

```diff
@@ -0,0 +1,250 @@
+```javascript
+{props.user.github && (
+  <a
+    href={props.user.github}
+    className={styles.card}
+  >
+    <h3>Visita su GitHub &rarr;</h3>
+    <p>Estoy seguro que está lleno de proyectos y tiene mucho verde.</p>
+  </a>
+)}
+```
+
+¡Reto! ¿Puedes hacer lo mismo para el link de su sitio web? La variable es: `{props.user.website}`.
+
+No vale mirar, la solución es:
+
+```javascript
+{props.user.website && (
+  <a
+    href={props.user.website}
+    className={styles.card}
+  >
+    <h3>Visita su sitio web &rarr;</h3>
+    <p>Su pequeño rincón en Internet, ¡quién sabe qué encontrarás!</p>
+  </a>
+)}
+```
+
+<img src="https://cloud-a1hqcjanz.vercel.app/ezgif-7-3455d319b9c1.gif" width="450" alt="¡Funciona!">
+
+### Mejoremos nuestro sitio✨
+
+¡Hurra! Acabas de hacer algo espectacular, ¡hágamoslo aún mejor!
+
+Igual y te diste cuenta que algunas veces obtenemos un bot o un usuario inactivo, ¡queremos personas que podamos conocer! Todos amamos a Orfeo, pero no es una persona real y tampoco lo es `@wb_bot_a01a9mk4fqw`.
+
+<img src="https://cloud-39fyb55qc.vercel.app/screenshot_2020-09-26_at_5.58.05_pm.png" width="450" alt="¡Funciona!">
+
+Cambiemos esto filtrando a los usuarios.
+
+¿Cómo podemos determinar si un usuario es activo? La manera más fácil es checar si han publicado en Scrapbook. Cada objeto de usuario tiene un campo llamado `updatesCount` que nos dice cuántas publicaciones han hecho. 
+
+Para filtrar en Javascript podemos usar `.filter`: le damos un array, algunos criterios y nos retorna un nuevo array con los elementos que cumplieron con los criterios.
+
+En nuestro caso necesitamos agregar este (↓) código justo abajo de la línea donde obtuvimos nuestros usuarios.
+
+```javascript
+users = users.filter(u => u.updatesCount != 0)
+```
+
+Recarga la página, ¡ya no deberíamos obtener gente inactiva ni bots!
+
+¡Nuestro sitio está finalizado! Esperamos te sientas orgulloso porque ahora es momento de que tomes crédito de tu creación :D
+
+Si scrolleas hacia abajo encontrás `@yourname`, ¡reemplázalo con tu nombre!
+
+### ¡Tiempo de Hackear!
+
+Un gran dato conlleva una gran oportunidad. Hay mucho que podemos hacer con estos datos, ¡así que experimenta con ellos!
+
+Si no sabes qué hacer, aquí hay algunos ejemplos de siguientes pasos:
+
+* Usando el CSS personalizado de Scrapbook, ¡[esta versión del proyecto](https://meet-hackclub.vercel.app) hace cada perfil más personal!
+* ¿Quieres obtener un usuario específico? Sam Poder construyó una versión que te permite hacerlo si agregas un nombre de usuario a la URL, [prueba obteniendo a Sam aquí](https://meet-hackclub.vercel.app/sampoder). 
+* ¿Qué otros datos podemos conseguir? Podemos obtener una publicación del Scrapbook de alguien, ¡tal como [Sam hizo aquí](https://meet-hackclub.vercel.app/post-ver)!
+
+¿Estás haciendo algo cool? ¡Fascinate! ¡Compártelo en [#ship](https://hackclub.slack.com/archives/C0M8PUPU6/) en Slack y taggea a [@sampoder](https://hackclub.slack.com/archives/DT08DHJKF/)!
```

Comment:
Hi @davidballezaa! I would suggest to translate "Awesome" as `Asombroso` , besides that everything looks great!

Reply:
Done, thank you!

Reply:
Awesome! Thanks for the help!